### ISSUE TITLE
Customizable frontend

### ISSUE THREAD

**Issue by 0xCAP** [MeStrak/catkin#240]
**Is your feature request related to a problem? Please describe.**
No, it's about a feature I often miss on other apps.

**Describe the solution you'd like**
I'd like the app's front-end to be highly customizable.

**Describe alternatives you've considered**
I'd like to let users pick and choose colors, background images and even fonts.

**Comment by MeStrak** [comment]
Hi @0xCAP, glad to see you were able to log in in the end!

For sure I want to make the front end customisable by picking background images from unsplash, etc.

I'm not ready to work on it yet because there are still a lot of basic features missing in the app, but it will come :)

Do you have any specific use case in mind for this (to embed in another website, or display on a large screen for example), or is it mainly just to configure the app how you like it?

**Comment by 0xCAP** [comment]
No specific use case in my mind, I only think it'd be a neat feature to have!

**Comment by MeStrak** [comment]
Ok :)

I'm thinking of setting it at the kanban board level (soon it will be possible to have multiple kanban boards). The idea is that the kanban board creator sets a background image + colour scheme for that board which will apply to any user viewing it.

What do you think?

**Comment by 0xCAP** [comment]
Yes, sound perfect to me. It'd be cool too to be able to export the board's current visual settings!
